releasenotes/notes/fix-isometry-rust-adf0eed09c6611f1.yaml

```
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fix a bug in :class:`~.library.Isometry` due to an unnecessary assertion,
+    that led to an error in :meth:`~.library.UnitaryGate.control`
```

This will make the display text `UnitaryGate.control` and link to the same thing which will probably be a bit simpler:

```suggestion
    that led to an error in :meth:`.UnitaryGate.control`
```